src/main/java/net/fabricmc/loader/discovery/ModResolver.java

```diff
@@ -69,6 +69,7 @@
 			.build()
 	);
 	private static final Map<String, List<Path>> inMemoryCache = new ConcurrentHashMap<>();
+	private static final Map<URL, String> readableNestedJarPaths = new ConcurrentHashMap<>();
```

Comment:
Should be `Map<String,` rather than `Map<URL,`for the same reasons as in https://github.com/FabricMC/fabric-loader/pull/278